package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "3.4.0",
+	"version": "3.6.0",
```

Comment:
Skipped 3.5.0 due to last minute decision to include Quick Setup changes https://github.com/adobe/aio-cli-plugin-api-mesh/pull/145